docs/recipes/pug.md

```diff
@@ -34,27 +34,47 @@ $ npm install --save-dev gulp-pug
 Add this task to your `gulpfile.js`, it will compile `.pug` files to `.html` files in `.tmp`:
 
 ```js
-gulp.task('views', () => {
-  return gulp.src('app/*.pug')
+function pug() {
+  return src('app/*.pug')
     .pipe($.plumber())
     .pipe($.pug({pretty: true}))
-    .pipe(gulp.dest('.tmp'))
-    .pipe(reload({stream: true}));
-});
+    .pipe(dest('.tmp'))
+    .pipe(server.reload({stream: true}));
+}
 ```
 
 We are passing `pretty: true` as an option to get a nice HTML output, otherwise Pug would output the HTML on a single line, which would break our comment blocks for wiredep and useref.
 
-### 3. Add `views` as a dependency of both `html` and `serve`
+### 3. Add `pug` task to `server` and `build` process
 
 ```js
-gulp.task('html', ['views', 'styles', 'scripts'], () => {
+if (isDev) {
+  serve = series(clean, pug, parallel(styles, scripts, fonts), startAppServer);
+} else if (isTest) {
+  serve = series(clean, pug, scripts, startTestServer);
+} else if (isProd) {
+  serve = series(build, pug, startDistServer);
```

Comment:
The `build` task is already running the task, is there a reason to run it here as well?

```suggestion
  serve = series(build, startDistServer);
```